.github/workflows/go.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Test
         shell: bash
-        run: ./build_and_test.sh
+        run: ./build_and_test.sh -v
```

Comment:
What's the point of this? 
build_and_test.sh doesn't take `-v` AFAICT.

Reply:
it does take `-v` :)
it'll pass it into the go commands, so it will print the name of the test it's running, which will help us figure out exactly which test is spuriously failing at `leveldb closed`